### PR TITLE
Add authorised fleet property

### PR DESCRIPTION
**Pull request by FlavianEng**

### CODE REVIEW

**Comment by FlavianEng** [comment]
Task linked: [CU-3f7perj Add authorized fleet property](https://app.clickup.com/t/3f7perj)

File: src/engine/gateways/gameGateway.ts [hunk 1]
```diff
@@ -177,8 +177,14 @@ export class GameGateway implements OnGatewayConnection {
 
     socket.join(String(instance.id));
 
-    const room: RoomData<GamePlayer> = {
-      data: baseGameSettings.firstPlayer,
+    const room: RoomData<{
+      players: GamePlayer[];
```
Comment (MASBERNATQ) [inline src/engine/gateways/gameGateway.ts:181]: Créer un type

File: src/tests/game-instance.service.spec.ts [hunk 2]
```diff
@@ -559,10 +557,7 @@ describe('GameInstanceService', () => {
     service.fleets = fleets1();
     service.players = players1();
     service.players.push(loggedPlayer1());
-    service.fleets['drakenline_0'] = [
-      validBoatPlacement3(),
-      validBoatPlacement3(),
-    ];
+    service.fleets['drakenline_0'] = [validFrigate(), validFrigate()];
```
Comment (MASBERNATQ) [inline src/tests/game-instance.service.spec.ts:560]: 👀

**Review by MASBERNATQ — APPROVED** [review]

**Review by DrakenLine — APPROVED** [review]

**Pull request merged**